packages/preferences/src/browser/preferences-decorator.ts

```diff
@@ -45,37 +45,39 @@ export class PreferencesDecorator implements TreeDecorator {
         if (!this.preferences) {
             this.preferences = preferences;
         }
-        this.preferencesDecorations = new Map(preferences.map(m => {
-            const preferenceName = Object.keys(m)[0];
-            const preferenceValue = m[preferenceName];
-            const storedValue = this.preferencesService.get(preferenceName, undefined, this.activeFolderUri);
-            const description = this.getDescription(preferenceValue);
-            return [preferenceName, {
-                tooltip: this.buildTooltip(preferenceValue),
-                captionSuffixes: [
-                    {
-                        data: `: ${this.getPreferenceDisplayValue(storedValue, preferenceValue.defaultValue)}`
-                    },
-                    {
-                        data: ' ' + description,
-                        fontData: { color: 'var(--theia-descriptionForeground)' }
-                    }]
-            }] as [string, TreeDecoration.Data];
-        }));
+        if (preferences) {
```

Comment:
what is a guard for? `preferences` are always defined

Reply:
@NicholasStenbeck, I think you introduced this check. Can you shed some light here?